portfolio/src/main/webapp/index.html

```diff
@@ -1,109 +1,150 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <!-- Roboto font link -->  
-    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap" rel="stylesheet">
-    <!-- Arvo font link -->
-    <link href="https://fonts.googleapis.com/css2?family=Arvo:ital@1&display=swap" rel="stylesheet">
-    <meta charset="UTF-8">
-    <title>Andrew's Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-  </head>
-  <body>
-    <!-- Wrapper for all content -->
-    <div id="content">
-        <!-- Wrapper for header items -->
-        <div id="header">
-            <div id="avatarImg">
-                <img src="/images/avatarAndrew.jpeg" alt="Avatar" id="avatar">
-            </div>
 
-            <div id="title">
-                <h1>Andrew Wiedenmann</h1>
+<head>
+	<!-- Roboto font link -->
```

Comment:
Prefer 2 spaces over tabs for indentation

Reply:
done

Reply:
done

Reply:
Is it github's rendering or are the indents much larger than I expected w/2 spaces?

Reply:
Must be a github thing. I just pulled it down and it renders as two spaces locally.

Reply:
Ok I made a new branch, rebased it with andrewDevelopment, and the html has 2-space tabs. I am 99% sure this is a github issue. If that doesn't sound right to anyone let me know!

Reply:
SGTM!